### DISCUSSION TITLE
R68s 的流量监控无法正常工作，常常出现异常的流量读数

### DISCUSSION THREAD

**Discussion by Lixuhuilll** [immortalwrt/immortalwrt#986] [General]
### Describe the bug

如图所示，流量监控功能检测到的网络流量总是少于实际应有的流量，且不时出现异常的流量尖峰，关闭软件流量分载也无改善，旁边的速率检测功能也无法工作，所有设备都只显示不到1KB/s的网速。
![image](https://user-images.githubusercontent.com/88485037/224059731-3cf352d5-b459-4515-8973-52f165e5718b.png)

### ImmortalWrt version

r25487-8df50f3b41

### ImmortalWrt target/subtarget

rockchip/armv8

### Device

Lunzn FastRhino R68S

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by Heisenberg5201314** [comment]
有交流群那种吗  各位大佬们
